### PR TITLE
DOCS Clarify CVE process

### PR DESCRIPTION
**Pull request by chillu**

### CODE REVIEW

File: docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md [hunk 1]
```diff
@@ -113,7 +113,9 @@ is available, or within a reasonable period of time of such a release.
    * Use the [CVSS Calculator](https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator) to determine the issue severity
    * Once the issue is confirmed, [request a CVE identifier](https://cveform.mitre.org/) under the security@silverstripe.org contact email (see "Acknowledgement and disclosure").
    * Once a CVE has been assigned, respond to issue reporter and add it to the Github issue 
-   * Clarify who picks up owns the issue resolution (assign in Github)
+   * Clarify who picks up owns the issue (assign in Github).
```
Comment (robbieaverill) [inline docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: Picks up and owns?

File: docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md [hunk 2]
```diff
@@ -203,6 +208,23 @@ It's important that you re-run `satis build` step after EVERY change that is pus
 each release, if making multiple releases.
 </div>
 
+### Detailed CVE and CVSS Guidance
+
+ * In the [CVE Request Form](https://cveform.mitre.org/), we follow certain conventions on fields:
+   * **Vendor of the product(s):** SilverStripe
+   * **Affected product(s)/code base - Product:** Composer module name (e.g. `silverstripe/framework`).
+     Indirectly affected dependencies of the module should not be listed here.
+   * **Affected product(s)/code base - Version:** Use composer constraint notation,
```
Comment (robbieaverill) [inline docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: Composer*

File: docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md [hunk 3]
```diff
@@ -203,6 +208,23 @@ It's important that you re-run `satis build` step after EVERY change that is pus
...
+     with one entry per major release line.
+     Example for an issue affecting all framework releases: First line `^3.0`, second line `^4.0`.
+     We don't know the target release version at this point, so can't set an upper constraint yet.
+     Should include versions which are unsupported.
```
Comment (robbieaverill) [inline docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: Might be worn clarifying this. It should include all affected versions, including any unsupported release lines. This half implies it should always affect unsupported releases.

Comment (madmatt) [reply docs/en/05_Contributing/05_Making_A_SilverStripe_Core_Release.md]: That implies you need to test it on all versions that are unsupported then...

I guess if it's easy enough to see the issue, you could just look back at v3.0.0 and say 'yep, it was a problem then, so all versions of 3 are affected'?

Or we just go the easy route and say "everything before this version of <major>.<minor>.<patch> is affected"

**Pull request merged**